storage/innobase/lock/lock0lock.cc

```diff
@@ -3828,6 +3828,20 @@ lock_table_enqueue_waiting(
 	}
 
 #ifdef WITH_WSREP
+	/*
+	  MDEV-32738:
```

Comment:
Actually, a BF-transaction is really waiting on the lock and it will be aborted only if it's in the waiting state (it happens below in `lock_table_create()` call). But I agree, the comment is a bit misleading. I will change to mention a conflicting lock.